app/assets/stylesheets/application.css.scss

```diff
@@ -34,15 +40,59 @@ body {
 
 .row {
   color: #461B7E;
+  max-width: 1200px;
+}
+
+.panel {
+  overflow: hidden;
+  max-width: auto;
+  text-align: center;
+}
+
+
+// Default medium-gray color
+$medium-gray: #cacaca;
+// Default global padding
+$global-padding: 1rem 3rem;
+// Global value used for all elements that have a border radius.
+$global-radius: 5px;
+
+// Setting the form border color
+$form-border-color: $medium-gray;
+// Setting the form padding
+$form-padding: $global-padding;
+// Setting the form border radius
+$form-radius: $global-radius;
+
+.log-in-form {
+  border: 1px solid $form-border-color;
+  padding: $form-padding;
+  border-radius: $form-radius;
+  background-color: white;
+}
+
+.page-title {
+  padding-left: 3rem;
 }
 
 /*** for About_Michelle ****/
+.michelle-background-image {
+  background: image-url('background_picture.jpg');
+  padding: 10px 5px 10px 5px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px 5px` instead of `10px 5px 10px 5px`